generators/client/templates/angular/src/main/webapp/app/entities/user/user.model.ts.ejs

```diff
@@ -30,3 +30,7 @@ export class User implements IUser {
     public login: string,
   ) {}
 }
+
+export function getUserIdentifier(user: IUser): <%= idType %> | undefined {
+  return user.id;
+}
```

Comment:
Why adding this function, why not using just `user.id`?

Reply:
I've added to entities too.
- I have a PR for composite ids that will change every `primaryKey.name` reference.
- Can be reused at `trackEntityByIdentifier` functions.

This should make easier for the upcoming PR, anyway if you prefer to keep references, I'm ok with it.

Reply:
OK, good point about keeping User entity similar to other entities and this way simplify generator, so this is good to go.